scipy/stats/_covariance.py

```diff
@@ -89,6 +90,34 @@ def from_diagonal(diagonal):
         does not lie in the span of the columns of the covariance matrix. The
         convention taken here is to treat the inverse square root of
         non-positive elements of :math:`d` as zeros.
+
+        Examples
+        --------
+        Prepare a symmetric positive definite covariance matrix ``A`` and a
+        data point ``x``.
+
+        >>> import numpy as np
+        >>> from scipy import stats
+        >>> rng = np.random.default_rng()
+        >>> n = 5
+        >>> A = np.diag(rng.random(n))
+        >>> x = rng.random(size=n)
+
+        Extract the diagonal from ``A`` and create the ``Covariance`` object.
```

Comment:
Is this intentional that you formatted `Covariance` without linking? I suppose since you have it with single backticks above.

Reply:
Nope. Should be single.
```suggestion
        Extract the diagonal from ``A`` and create the `Covariance` object.
```